README.md

```diff
@@ -12,26 +12,78 @@ Vaporetto is a fast and lightweight pointwise prediction based tokenizer.
 
 This repository includes both a Rust crate that provides APIs for Vaporetto and CLI frontends.
 
-The following examples use [KFTT](http://www.phontron.com/kftt/) for training and prediction data.
+### Try Word Segmentation
 
-### Training
+This software is implemented in Rust. Install `rustc` and `cargo` following [the documentation](https://www.rust-lang.org/tools/install) beforehand.
 
+Vaporetto provides two ways to generate tokenization models:
+
+#### Convert KyTea's Model
+
+The first is the simplest way, which is to convert a model that has been trained by KyTea.
+First of all, download the model of your choice from the [KyTea Models](http://www.phontron.com/kytea/model.html) page.
+
+We chose `jp-0.4.7-5.mod.gz`:
+```
+% wget http://www.phontron.com/kytea/download/model/jp-0.4.7-5.mod.gz
+```
+
+Each model is compressed, so you need to decompress the downloaded model file like the following command:
+```
+% gunzip ./jp-0.4.7-5.mod.gz
+```
+
+To convert a KyTea model into a Vaporetto model, run the following command in the Vaporetto root directory.
+If necessary, the Rust code will be compiled before the conversion process.
+```
+% cargo run --release -p convert_kytea_model -- --model-in path/to/jp-0.4.7-5.mod --model-out path/to/jp-0.4.7-5-tokenize.model.zstd
+```
+
+Now you can perform tokenization. Run the following command:
 ```
-% cargo run --release --bin train -- --model ./kftt.model.zstd --tok ./kftt-data-1.0/data/tok/kyoto-train.ja
+% echo '火星猫の生態の調査結果' | cargo run --release -p predict -- --model path/to/jp-0.4.7-5-tokenize.model.zstd
 ```
 
-### Prediction
+The following will be output:
+```
+火星 猫 の 生態 の 調査 結果
+```
+
+#### Train Your Own Model
+
+The second way, which is mainly for researchers, is to prepare your own training corpus and train your own tokenization models.
+
+Vaporetto can train from two types of corpora: fully annotated corpora and partially annotated corpora.
+
+Fully annotated corpora are corpora in which all character boundaries are annotated with eather token boundaries or internal positions of tokens.
```

Comment:
```suggestion
Fully annotated corpora are corpora in which all character boundaries are annotated with either token boundaries or internal positions of tokens.
```
Thanks! I found a minor misspelling. Otherwise, no problem.